### ISSUE TITLE
Core Devs Call: Confirm consensus for Aztlán Upgrade (Yingchun Edition)

### ISSUE THREAD

**Issue by bobsummerwill** [ethereumclassic/ECIPs#215]
### ETC Core Devs Call - Confirm consensus for Aztlán Upgrade (Yingchun Edition)

When: **Thursday, December 5, 2019, 1pm UTC**, 60 minutes max.

Where: **Ethereum Classic Discord** https://discord.gg/dwxb6nf `#ecips` channel. Will use/create a voice channel _ad hoc._

Meeting Coordinator: @developerkevin.

### Agenda

- Aztlán Upgrade (Yingchun Edition) ([ECIP-1061](https://ecips.ethereumclassic.org/ECIPs/ecip-1061)) is in _"Last Call"_ status with review ending on 19th Dec 2019.
  - Establish that we have new consensus on ECIP-1061, after several days of crisis.
  - No finger-pointing, blame or post-mortem in this specific meeting.   It would make a lot of sense to do a post-mortem on the last few days, and anybody is welcome to create an issue to start that process, or to publish their own post-mortem, etc.    Just out of scope HERE.
  - No discussion of #221 and #224 or other process clarifications.   Similar to post-mortem, these have value and when we have fully-formed ECIPs we should certainly discuss those.   Just out of scope HERE.

This is the full scope of the agenda.  We won't add any items to this call.  Laser focus.

The aim of the meeting is to calm everybody's nerves and to get back on track.   We should record the audio.   Multiple people should take minutes.   The outcome should be reported on.

**Comment by TheEnthusiasticAs** [comment]
I am for it. It is fair to both (1061; 1072) particiapants

**Comment by sorpaas** [comment]
Given there's just too much politics in Ethereum Classic currently, I'll be openly oppose this meeting and the result of this meeting. The meeting does not represents the Ethereum Classic community and we will be treating this as a contentious hard fork.

**Comment by shanejonas** [comment]
there should only be one ecip: 1061, and we can update it to reflect the changes to the hard fork. Not sure why a second one was created. Doing so does not follow any ECIP process.

**Comment by meowsbits** [comment]
I'm OK with a DDoS-able process and have no issues with duplicates.

Because the bickering over duplicate documents at this point _is_ the DDoS.

**Comment by soc1c** [comment]
As stated in https://github.com/ethereumclassic/ECIPs/issues/175#issuecomment-552188293 I'm not available for a call on Dec/5.

Also, I believe there is a process in place for these cases and we should just stick to the ECIP-1000 process which allows editing draft ECIPs and therefore does not require ECIPs 1072, ..73, ..74, and ..75. Everything went downhill when Wei tried to prevent me from updating the draft ECIP 1061.

We always updated working-specs as we did for Atlantis and Agharta. I don't see why we should now change the process for Aztlan.

On the last call, we agreed on Aztlan moving to "last call" in the Yingchuan flavor as proposed by Wei without 1884. We did not agree on withdrawing 1061 which is the original Aztlan ECIP for months now. Having another call is superfluous in my humble opinion.

**Comment by bobsummerwill** [comment]
See also:
https://github.com/ethereumclassic/ECIPs/issues/217#issuecomment-559861432

**Comment by bobsummerwill** [comment]
RE: "Too much politics"

https://en.wikipedia.org/wiki/Politics

"Politics is a set of activities associated with the governance of a country, state or an area. It involves **making decisions** that apply to group of members."

**Comment by bobsummerwill** [comment]
Wei is just closing issues without justification.
Reopening

**Comment by sorpaas** [comment]
This call will be useless now as we do have consensus.

**Comment by bobsummerwill** [comment]
You are not attending.
You have made that clear.

I am proposing this meeting.
It is not your place to keep closing it down.

**Comment by bobsummerwill** [comment]
"This call will be useless now as we do have consensus."

This is not your decision to make unilaterally, @sorpaas.
Please do not close this a third time.
I will discuss with everyone, and we may or may not move forward.

But you are not helping yourself by keeping shutting this down.

**Comment by husainfazel** [comment]
```Hey everyone. First of all, I want to post an apology for my actions in the last few hours. I have certainly been hugely over-reacting and caused a lot of unnecessary fusions. That is my fault. I think there are few things I want to say and actions to correct.

• Regarding 1061 vs 1072, I won't back down on my position that it's indeed Afri's fault not following the ECIP process in the first place. However, given that it's also largely my fault that I over-reacted and was being too aggressive in the past two days, I'll agree to move forward with 1061, and withdraw 1072.
• During the last minute review I done on the Aztlan specification we currently have, I indeed find a few additional concerns that will have some impacts. From technical point of view, my recommendations would be we hold off Aztlan until account versioning is implemented. However, given that in the core dev meetings we have accepted the Aztlan hard fork, I will not oppose the hard fork. In the mean time, I may still make those technical concerns public, even though it's hard to fix.
• I have not yet decided how much engagement I'll devote to Ethereum Classic in the future. However, I'll not cancel any of the current ECIP initiatives. In particular, I strongly think rotations of meeting coordinators will be an important process we should implement, starting with Yaz and Afri.

Good luck everyone. And have a nice weekend!
```

@sorpaas why do you keep closing this issue? I think you and Bob need to have a private chat and stop playing silly buggers. The call doesn't need to happen, it probably won't happen... but let's leave it to Bob to handle that. 

Your comments in the Discord were really mature but what you're doing here is ridiculous and undermines all the bridge building you were magnanimous enough to start a few hours ago.

**Comment by sorpaas** [comment]
@bobsummerwill @husainfazel Please stop spamming the ECIP repo.

> The call doesn't need to happen, it probably won't happen.

As you have correctly stated. It's Bob's job to provide justifications if he really wants to keep this issue open, but apparently he hasn't yet.

**Comment by bobsummerwill** [comment]
Reopening as per https://github.com/ethereumclassic/ECIPs/pull/220#issuecomment-560141304.

**Comment by sorpaas** [comment]
Not sure if I have time to join this meeting, but I agree to switch consensus from last meeting, which was ECIP-1072, to the new one, ECIP-1061.

**Comment by bobsummerwill** [comment]
I don't think it needs to be a long call.

@soc1c said he is at a conference this Thursday, but maybe he can "sneak out" for 15 mins?  Afri?

Even if neither of you are there, I think we need this meeting.    Perhaps it is even better WITHOUT both of you there, because the meeting is really for the good of people on the outside looking in and thinking "WTF!   So are we cool here?   Are we in the middle of a civil war?   Is there doing to be a contentious fork here?   I thought we were catching up on ETH protocol and having a majestic Phoenix from the flames moment.   Is everything just going to be a disaster instead?"

The answer to all of those is NO.    But we could use that affirmation in a meeting, I think.   Recorded.  Minuted.  Reported on.

Maybe @YazzyYaz can host?   Or I can host?   See also #233.

**Comment by sorpaas** [comment]
@bobsummerwill I would advise against you or @YazzyYaz hosting this meeting because ETC Coop's position is not neutral this time. It would be much better choosing one from this list https://github.com/ethereumclassic/ECIPs/issues/209

**Comment by sorpaas** [comment]
Of course, I wouldn't mind it as a last resort, if you indeed cannot find anyone else to be the coordinator.

**Comment by bobsummerwill** [comment]
@developerkevin?  @phyro?   Can either of you host, please?

@sorpaas Looks like we were editing on top of each at the top of this.   I have just done a rewrite.   Does this wording work for you?

Ie. scope is ONLY to clarify that we have consensus on 1016 as it NOW exists - including with you as co-author, and with Apache 2.0 licensing now.

**Comment by phyro** [comment]
@bobsummerwill unfortunately, lately I've not been able to join the calls due to being busy with other stuff and very likely won't be able to attend this one as well.

**Comment by bobsummerwill** [comment]
Too bad, @phyro.  Thanks for the reply.

**Comment by TheEnthusiasticAs** [comment]
What about @mikeyb ?

**Comment by bobsummerwill** [comment]
Or YOU, @TheEnthusiasticAs!

**Comment by TheEnthusiasticAs** [comment]
I would gladly, but I have a general language barrier in english speaking at the moment (not using it in everyday life). I understand the spoken, but, when it gets more (software) technical/topic specific I need more time for understanding. It is not my native language. I am working on it ;-)

**Comment by developerkevin** [comment]
> @developerkevin? @phyro? Can either of you host, please?

@bobsummerwill Yeah that's not a problem!

**Comment by bobsummerwill** [comment]
Thanks, @developerkevin!

So I have added you in as the host.   Please could you publicize the meeting too?    The BEST way now would be for you to author a blog post within the new EthereumClassic.org website.

That means authoring a single MD file, like these:

https://github.com/ethereumclassic/ethereumclassic.github.io/commit/c0eb2658e74cdd6b2ebb56d12aff6bfbd2770c51

https://github.com/ethereumclassic/ethereumclassic.github.io/commit/46622f7943f51646d1dbee2480a3ad84749f2599

https://github.com/ethereumclassic/ethereumclassic.github.io/commit/e2c9f38b2b17ce526255cb9fcebb0d665aa3fec0

Into ...

https://github.com/ethereumclassic/ethereumclassic.github.io

So you are literally just dropping a new MD file into 'pages/blog' folder of 'source' branch.
If you want to add images too, just drop (preferably PNG) images into 'pages/assets'

and then you can reference them in the MD files like this:

`![Bob and Yaz](../../assets/bob_and_yaz.jpeg)`

To preview you can run Node locally, as per the instructions in the readme, or you can just submit your PR, and wait on the automated builds to give you a preview URL.   @hitchcott can, no doubt, help you with any issues, or just ask on the #ethereumclassicwebsite channel on Discord.

**Comment by bobsummerwill** [comment]
Or if that is too much hassle to do today/tomorrow then just do a Medium article for now, and we can move it over later.

Main TODO is get the message out that we are having a call, and the purpose of that call is PURELY to establish that we do have consensus on 1061 and that we are confirming that everyone is cool with 1061, that we don't have any kind of community split, or contentious fork situation ongoing, and that everything is on track for Aztlan in March.

NOT IN SCOPE for this meeting is doing any kind of post-mortem on events.
NOT IN SCOPE for this meeting is going through proposals on changes to ECIP process to avoid this kind of situation again.

Both of those are too fresh and too "in-motion" to justify talking about them on a call.    Instead we can use the Github ECIP workflow for those, and then come back and have another call when there is something concrete being proposed.

**Comment by sorpaas** [comment]
@bobsummerwill The later shouldn’t have calls for decision. Decisions on ECIP process should be reached in written form. Meetings are exclusively reserved for hard forks.

**Comment by bobsummerwill** [comment]
Great.
Even easier, @sorpaas.

**Comment by developerkevin** [comment]
I'll get on that blog post today and finish it into tomorrow @bobsummerwill

**Comment by developerkevin** [comment]
I'll get on that blog post today and finish it into tomorrow @bobsummerwill

**Comment by bobsummerwill** [comment]
Great, @developerkevin!   Please just shout out on Discord in the ethereumclassicwebsite channel if you need some help on the MDs.   Or just Medium post.  Whatever.   We just need it announced ASAP.

**Comment by soc1c** [comment]
What's the outcome of this? Any notes?

**Comment by husainfazel** [comment]
They didn't record it due to technical difficulties but essentially it was a short call where everyone agreed to re-establish consensus on ECIP-1061 in its current state, and that it SHOULD be in Last Call status.

Link for the discussion of the call on Discord
https://discordapp.com/channels/223674353001168906/552567121046011905/652137098911940628

**Comment by soc1c** [comment]
ok
